pkg/base1/test-series.html

```diff
@@ -523,6 +524,11 @@ <h2 id="qunit-userAgent"></h2><ol id="qunit-tests"></ol>
     });
 });
 
+/* make sure that $.now() returns a constant value to avoid races */
```

Comment:
Lets use the term "mock" here to make it clear this is the usual testing approach.